src/timeseries.py

```diff
@@ -19,10 +19,20 @@ def get_dataset_time_series(org_path=None, theme=None, theme_profile=None, organ
                                                    org_path=org_path,
                                                    theme=theme,
                                                    theme_profile=theme_profile,
-                                                   organization_id=organization_id)
+                                                   organization_id=organization_id,
+                                                   series_field="dcatRecord.http://purl.org/dc/terms/issued.value")
+    return TimeSeriesResponse(es_time_series)
+
+
+def get_concept_time_series(org_path=None, organization_id=None) -> TimeSeriesResponse:
+    es_time_series = elasticsearch_get_time_series(report_type=ServiceKey.CONCEPTS,
+                                                   org_path=org_path,
+                                                   organization_id=organization_id,
+                                                   series_field="harvest.firstHarvested")
```

Comment:
add field to EsMappingClass(f.eks FIRST_HARVESTED = "harvest.firstHarvested or FROM_ES_TIME_SERIES)